packages/core/src/App/Containers/Layout/header/traders-hub-header-wallets.tsx

```diff
@@ -113,17 +119,19 @@ const TradersHubHeaderWallets = observer(() => {
                         <div className='traders-hub-header__menu-right--items--notifications'>
                             <ShowNotifications />
                         </div>
-                        <Popover
-                            classNameBubble='account-settings-toggle__tooltip'
-                            alignment='bottom'
-                            message={<Localize i18n_default_text='Manage account settings' />}
-                            should_disable_pointer_events
-                            zIndex={'9999'}
-                        >
-                            <BinaryLink className='traders-hub-header__setting' to={routes.personal_details}>
-                                <Icon icon='IcUserOutline' size={20} />
-                            </BinaryLink>
-                        </Popover>
+                        {isTabletOs ? (
+                            accountSettings
+                        ) : (
+                            <Popover
+                                classNameBubble='account-settings-toggle__tooltip'
+                                alignment='bottom'
+                                message={<Localize i18n_default_text='Manage account settings' />}
+                                should_disable_pointer_events
+                                zIndex={'9999'}
```

Comment:
just a small nit-pick, i know it was like this before :):
```suggestion
                                zIndex='9999'
```